config/development.yml.erb

```diff
@@ -35,8 +35,9 @@ db_writer: 'mysql://root@localhost/'
 
 custom_error_response: false
 
-# Used for the exporting teacher enrollment data to a gsheet
-enrollments_summer_2020_gsheet_key:
+# Used for exporting the workshop data to a gsheet
+applications_2022_2023_gsheet_key: !Secret
+gdrive_export_secret: !Secret
```

Comment:
what is this `gdrive_export_secret` secret used for?

Reply:
It authorizes the gdrive session when running the script so that the account can then go to the file and write to it (if the file gives the account permission)